src/pages/signup/components/SignupEmailVerify.tsx

```
@@ -26,9 +38,51 @@ const SignupEmailVerify = ({ onNext }: SignupProps) => {
+
+    try {
+      await verifyMutation.mutateAsync({
+        email,
+        type: 'SIGN_UP',
+      });
+      setTimeLeft(180);
+      setIsResendEnabled(false);
+      setVerifyNum('');
+      setErrorMessage('');
+      setSuccessMessage('');
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)